wee_slack.py

```diff
@@ -1407,6 +1407,7 @@ def destroy_buffer(self, update_remote):
             self.eventrouter.receive(s)
 
     def buffer_prnt(self, nick, text, timestamp=str(time.time()), tagset=None, tag_nick=None, **kwargs):
+        text = re.sub(r"```(?!\n)", "```\n", text)
```

Comment:
When the message changes, the message will be rendered again, which will overwrite this. To make this persist through message edits, you need to place this in the function called `render` instead.

This adds a newline at the end of the message if the message ends with ```. This should not happen.

Reply:
I think both comments are now addressed